argo-cd-apps/overlays/staging/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ resources:
   - ../../base/host
   - ../../base/member
   - ../../base/all-clusters
+  - ../../base/promotion
```

Comment:
My understanding is that if you reference the ApplicationSet here, it will be installed on both staging clusters, which as far as I understand is not what we want.
We probably want to install it just on rh01.

Reply:
@Roming22 Could you please help me out, how it handle it then